### ISSUE TITLE
Issues editing unicode character docs

### ISSUE THREAD

**Issue by dizzzz** [eXist-db/eXide#123]
from https://stackoverflow.com/questions/36371056/unable-to-properly-edit-unicode-charachters-in-exide:

> the backspace  and delete buttons seem to delete characters that are not under the cursor.
> - chrome 50 / macsox
> - safari 9.1 / maxosx
> 
> attached is a file with special characters

[test.xml.txt](https://github.com/wolfgangmm/eXide/files/203182/test.xml.txt)

**Comment by joewiz** [comment]
I think this is an inherent problem with the ace editor - see https://github.com/ajaxorg/ace/search?q=unicode&type=Issues&utf8=%E2%9C%93 and https://www.google.com/webhp?hl=en&q=site:github.com/ajaxorg/ace/+unicode.

And according to https://github.com/ajaxorg/ace/issues/3037#issuecomment-236442429: 

> This happens with many non-ascii characters (and the same error occurs in the github editor)
estimated chance of being fixed : 0.071%

So... close as "wontfix"?

**Comment by duncdrum** [comment]
I d say this is core browser territory. @joewiz Let me have a look with some of my own files, where I managed to work around the problem with some editors. Chances are thus is beyond eXide s influence.

**Comment by duncdrum** [comment]
yup font substitution in ace is playing tricks, even with files that the browser/ OS can actually handle. See also this [PR](https://github.com/ajaxorg/ace/pull/3137)  with a potential fix for BIDI languages, in essence overriding ace's font-selection. 

I vote not to close, so there is a place to track ace's development, and for users to see that its a known issues. 

Maybe a similar overriding of ace font selection could be implemented in exide based on ``@xml:lang``? People use eXide to work with xml in the browser. Both the browser and xml can handle Unicode, so eXide should to. This wouldn't solve the problem for ``.txt`` or ``.xq`` but could offer a workable compromise.
